src/plugins/mer/merbuildenginedetailswidget.cpp

```diff
@@ -64,6 +64,14 @@ MerBuildEngineDetailsWidget::MerBuildEngineDetailsWidget(QWidget *parent)
             + tr("Stop the virtual machine to unlock this field for editing.")
             + QLatin1String("</font>"));
 
+    m_ui->sshPortOccupiedLabel->setPixmap(Utils::Icons::CRITICAL.pixmap());
+    m_ui->sshPortOccupiedLabel->setToolTip(
+            QLatin1String("<font color=\"red\">")
+            + tr("This port may be already in use. "
+                 "In case of virtual machine connection failure, please, select another port.")
```

Comment:
Maybe: "Another application seems to be listening on this port. Choose another port unless you know what you are doing."

Reply:
I learned English by video games, so I think your variant is better.:) Fixed.